### ISSUE TITLE
Camperbot to ignore `@/all` thanks

### ISSUE THREAD

**Issue by BerkeleyTrue** [FreeCodeCamp/camperbot#76]

**Comment by sludge256** [comment]
So you don't want Camperbot to respond at all? It won't actually send brownies to everyone if that's what the concern is.
![thanks](https://cloud.githubusercontent.com/assets/10524623/14516500/8276d8c8-01b8-11e6-84ff-6a7172477aa6.PNG)

**Comment by BerkeleyTrue** [comment]
Yes. While it doesn't give anyone brownie points, it still hits the database.

**Comment by t3h2mas** [comment]
https://github.com/FreeCodeCamp/camperbot/blob/master/lib/bot/cmds/thanks.js#L38 Looks like a good starting point.

**Comment by SaintPeter** [comment]
Might make sense to make an array "Ignore List" in case we need to add other names to it, rather than just hard coding the "all".
